WordPress/WordPressTest/Test Data/TestPlans.swift

```diff
@@ -43,12 +43,17 @@ enum TestPlans {
         }
     }
 
-    static let allPlans = {
-        return [
-            TestPlans.free,
-            TestPlans.premium,
-            TestPlans.business
-            ]
-            .map({ $0.plan })
-    }()
+    var product: MockProduct {
+        let plan = self.plan
```

Comment:
Totally minor, but I did that to avoid executing `plan` each time I access a value when creating the `MockProduct`. Unnecessary?